jwe_decrypt.go

```diff
@@ -0,0 +1,45 @@
+package jwe
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+)
+
+func (jwe jwe) Decrypt(key interface{}) ([]byte, error) {
+
+	method := jwe.protected.Enc
+	if len(method) == 0 {
+		return nil, errors.New("no \"enc\" header")
```

Comment:
Can we extract the errors here in `ErrXX` values like the other files?

Reply:
Added